data/mesongenbuildreq.py

```diff
@@ -2,8 +2,11 @@
 import json
 import sys
 deps_json = json.loads(subprocess.run([sys.argv[1], "introspect", "--dependencies", "meson.build"], capture_output=True).stdout)
-deps = dict(sorted(zip([x['name'] for x in deps_json],[x['version'] for x in deps_json])))
-deps.pop('', None)
+unsorted_deps = dict(zip([x['name'] for x in deps_json],[x['version'] for x in deps_json]))
+unsorted_deps.pop('', None)
+deps = {}
+for lib in list(unsorted_deps.keys()) :
+    deps[lib] = unsorted_deps[lib]
```

Comment:
This isn't sorting anything, just making an unsorted copy.
```python
deps = dict(sorted(unsorted_deps.items(), key=lambda x: x[0])))
```
should created a sorted dictionary based on the unsorted deps above.

Although, looking at the code you don't even really need to make it a dictionary, you can just used the `deps = sorted(...)`, and then just iterate `for lib, version in deps:`